src/util/nodeSelectors.ts

```diff
@@ -190,6 +191,30 @@ export function selectWithLeadingDelimiter(...delimiters: string[]) {
   };
 }
 
+/**
+ * Creates a selector for multiple children of a node, useful for contiguous ranges. 
+ * When no arguments are pased, match on every child. 
+ * When items are passed with the inclusion predicate, only match on those items. 
+ * When items are passed with the exclusion predicate, only match on those items. 
+ * @param items A set of types to match against, either for inclusion or exclusion.
+ * @param predicateType Determines wither nodes within the items argument will be included or excluded.
+ * @returns A selection extractor
+ */
+export function childRangeSelector(predicateType: ChildNodeIncludePredicate = "inclusion", items: string[] = []) {
```

Comment:
Hmm. What was wrong with my original proposal of having two args, `typesToExclude` and `typesToInclude`?  Seems clearer